src/typings/ts-utils.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@openstax/ts-utils/dist/services/authProvider/browser';
```

Comment:
I think this type and the path having `dist` are required because Rex uses Node 14? It wouldn't work without these.